src/todoInput.js

```diff
@@ -0,0 +1,19 @@
+export function handleTodoInput(
+  $todoInput,
+  $todoList,
+  createTodoItemElement
+) {
+  const handleKeydownTodoInput = (event) => {
+    const todoText = $todoInput.element.value.trim();
```

Comment:
아래 코드처럼 작성하는 것과 무엇이 다를까요?

```suggestion
    const todoText = event.target.value.trim()
```

Reply:
결과적으로는 동일한 변화를 가져올 것 같지만

$todoInput.element은 해당 **요소에 직접 접근**할 때 사용합니다.

event.target은 이벤트 핸들러에서 발생한 **이벤트에 관련된 값을 가져올 때**, 사용합니다.

과정에서 돔을 직접 접근하는 것은 부담스럽겠지만, 이벤트의 값만 취하면 더 가볍지 않을까? 궁금합니닷!

Reply:
JavaScript만으로 개발을 하는 경우에는 둘중에 어떤 것을 사용해도 크게 상관은 없을 것 같은데요~
하지만 $todoInput.element와 같이 사용하면, $todoInput의 이름이 변경되는 경우에 2곳에서($todoInput 변수를 선언하는 쪽, 사용하는쪽) 변경이 이루어져야 하는 것도 있고 일반적으로 React나 Vue에서는 event.target.value 방식으로 많이 작성을 하기 때문에 event 객체를 이용해주면 더 괜찮을 것 같아요.

성능적인 측면에 있어서는 비슷할 것 같아요~

Reply:
아하 기존대로 하면 변경되는 곳에서 2번의 수정이 이루어져야하니 
말씀해주신 대로 event.target.value가 낫겠네요!!
오 감사합니당!